airflow/operators/python.py

```diff
@@ -171,7 +178,8 @@ def execute(self, context: Dict):
         self.op_kwargs = determine_kwargs(self.python_callable, self.op_args, context)
 
         return_value = self.execute_callable()
-        self.log.info("Done. Returned value was: %s", return_value)
+        if self.show_return_value_in_logs:
+            self.log.info("Done. Returned value was: %s", return_value)
```

Comment:
I wonder we should still log something here (just don’t show the return value). No strong opinion here though.

Reply:
Good point - I think it's good to write explicily "Done. Returned value not shown".

Reply:
I will add a branch to display the phrase that it is turned off(`"Done. Returned value not shown"` which is mentioned by @potiuk) when the option is turned off.